numpy/core/_multiarray_umath.py

```diff
@@ -0,0 +1,9 @@
+from numpy._core import _multiarray_umath
+from ._utils import _raise_warning
+
+_raise_warning("*")
```

Comment:
same here

Reply:
Removed - explained in the previous comment.